charts/orchestrator/templates/sonataflow-operator.yaml

```diff
@@ -18,12 +18,15 @@ spec: {}
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: logic-operator-rhel8
+  name: {{ .Values.sonataFlowOperator.subscription.name }}
   namespace: {{ .Values.sonataFlowOperator.subscription.namespace }}
 spec:
   channel: {{ .Values.sonataFlowOperator.subscription.channel }}
-  installPlanApproval: Automatic
+  installPlanApproval: {{ .Values.sonataFlowOperator.subscription.installPlanApproval }}
```

Comment:
Curious, why do we need to specify this field? Isn't `Automatic` good enough by default?

Reply:
Switched back to Automatic.